configfile.js

```diff
@@ -266,6 +266,7 @@ cfreader.ensure_enoent_timer = function () {
             })(files[i]); // END BLOCK SCOPE
         }
     }, 60 * 1000);
+    cfreader._enoent_timer.unref();
```

Comment:
Since moving config into a module (so npm plugins could depend on it) I've been moving more and more stuff to depend on haraka-config, with the eventual plan to deprecate the build-in conifg.js and configfile.js. We're a ways down that road and I don't recall right now what's left to do.